recipes/pyrost/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "pyrost" %}
+{% set version = "0.1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3d97c37623c038a3ee63c74b17bce331e80f72156c81d40f55a7e2d860848668
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+  host:
+    - {{ mpi }}
+    - openblas
```

Comment:
Rather than linking directly to `openblas` please use the blas wrappers if possible https://conda-forge.org/docs/maintainer/knowledge_base.html#blas

Reply:
Done, thank you for the suggestion